### PR TITLE
[Fix] Loot allocation for NM "No'Mho Crimsonarmor" and Benediction Petrify and doom.

### PR DESCRIPTION
**Pull request by hooksta4**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by zach2good** [comment]
Probably fine, but please remember that the PR title and the titles of each commit in the PR are all that's left after something is merged (and we scrape the PR titles to build our patch notes) .

"Fixes issue X" isn't a good descriptor of what you've changed here.

**Comment by hooksta4** [comment]
> Probably fine, but please remember that the PR title and the titles of each commit in the PR are all that's left after something is merged (and we scrape the PR titles to build our patch notes) .
> 
> "Fixes issue X" isn't a good descriptor of what you've changed here.

Sounds good. Will fix moving forward. Also I am not sure I am githubbing correctly... updated another issue (#822) but looks like it wrapped up in the same PR? Is that normal or do I need to do something else?

Thanks

**Comment by Xaver-DaRed** [comment]
Issue #891
Fixed title for you

**Comment by hooksta4** [comment]
> Issue #891 Fixed title for you

Thanks!

File: scripts/globals/abilities/benediction.lua [hunk 1]
```diff
@@ -36,6 +36,11 @@ ability_object.onUseAbility = function(player, target, ability)
         heal = maxHeal
     end
 
+    local power = 33 --chance to remove Doom. Basing off of Holy Water?
+
+    if (target:hasStatusEffect(xi.effect.DOOM) and power > math.random(1, 100)) then
```
Comment (Xaver-DaRed) [inline scripts/globals/abilities/benediction.lua]: Style guide: Outer parenthesis ( ) arent needed.

Also, IDK why fixing benediction is added here. Not a bad thing, but weird you didn't comment it.

EDIT: Ah, you did comment it.
Moving forward, do edit the first comment of the PR, for the lazy people like me.

**Review by Xaver-DaRed — CHANGES_REQUESTED** [review]

**Comment by Xaver-DaRed** [comment]
Issue #822

File: scripts/globals/abilities/benediction.lua (hunk 1 above)
Comment (hooksta4) [reply scripts/globals/abilities/benediction.lua]: I have no idea either... I pulled a new repository, made the adjustments and pushed to my fork. IDK maybe I should watch the instructional video for this

**Comment by TeoTwawki** [comment]
> > Probably fine, but please remember that the PR title and the titles of each commit in the PR are all that's left after something is merged (and we scrape the PR titles to build our patch notes) .
> > "Fixes issue X" isn't a good descriptor of what you've changed here.
> 
> Sounds good. Will fix moving forward. Also I am not sure I am githubbing correctly... updated another issue (#822) but looks like it wrapped up in the same PR? Is that normal or do I need to do something else?
> 
> Thanks

note for the future also preferred to break separate issue fixes into separate PRs as well.

Not going to make you redo this to split these up though. To do multiple PRs what you would do is create additional branches on your fork. In fact its strongly recommend to not put your changes into your base branch of your fork for exactly this reason: every time you create a new branch, you want to base it off your base branch (hence the name) so that it begins with only that content. Then each branch starts as a solid "base" to be built on top of.

https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell

File: scripts/globals/abilities/benediction.lua (hunk 1 above)
Comment (TeoTwawki) [reply scripts/globals/abilities/benediction.lua]: I suggest basing it on whatever Cursna does. Which might be the same as Holy Water, I didn't look. :stuck_out_tongue:

**Review by Xaver-DaRed — APPROVED** [review]
closes #822 
closes #891

**Pull request merged**